modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -34,6 +35,119 @@ admin_manual:release_notes.adoc
+
+The feature is disabled by default as it is not compatible with all use cases. To enable it, administrators can set `'file_storage.save_version_author' => true` in `config.php`.
+
+TIP: This feature is incompatible with S3 Object storage. Do not enable it if you're using S3 as a primary storage.
+
+=== Restrict Public Link Sharing to Certain Groups
+
+In certain scenarios it is not desired that all users of an ownCloud installation can create public links and share data publicly, respectively. To cover for this, ownCloud Server 10.9 comes with a new configuration option for administrators that allows restricting the creation of public links to users in certain groups (https://github.com/owncloud/core/pull/38980[#38980]).
+
+TIP: Existing public links will continue to work after introducing a restriction policy. The policy only applies to the creation of new public links.
+
+=== Ability to move a user's home folder
+
+In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows moving user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and takes care of updating all other related information. In addition, a new command `occ user:home` allows to list the storage locations of users.
+Administrators can find more information in the https://doc.owncloud.com/server/latest/admin_manual/configuration/server/occ_command.html#user-commands[ownCloud documentation about occ] and https://doc.owncloud.com/server/10.9/admin_manual/configuration/user/user_management.html#moving-the-user-home[about user management].
+
+=== Updated App Versions
+
+Find below a list of updated apps in comparison with the 10.8 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
+
+* https://marketplace.owncloud.com/apps/files_mediaviewer[Media Viewer] 1.0.5
+* https://marketplace.owncloud.com/apps/files_pdfviewer[PDF Viewer] 1.0.1
+* https://marketplace.owncloud.com/apps/files_texteditor[Text Editor] 2.4.1
+* https://marketplace.owncloud.com/apps/guests[Guests] 0.9.3
+* https://marketplace.owncloud.com/apps/oauth2[OAuth2] 0.5.2
+* https://marketplace.owncloud.com/apps/openidconnect[OpenID Connect] 2.1.0
+* https://marketplace.owncloud.com/apps/web[Web] 4.6.0
+* https://marketplace.owncloud.com/apps/windows_network_drive[Windows Network Drive] 2.1.0
+* https://marketplace.owncloud.com/apps/wopi[Microsoft Office Online] 1.5.1
+* https://marketplace.owncloud.com/apps/user_ldap[LDAP Integration] 0.16.0
+
+=== Other Notable Changes
+
+* The handling for inaccessible federated shares has been improved to better avoid removing temporarily unavailable shares. https://github.com/owncloud/core/pull/38474[#38474]
```

Comment:
```suggestion
* The handling of inaccessible federated shares has been improved to avoid removing temporarily unavailable shares. https://github.com/owncloud/core/pull/38474[#38474]
```